src/main/kotlin/calculrate/Calculator.kt

```diff
@@ -0,0 +1,21 @@
+package calculrate
+
+object Calculator {
+
+    fun calculate(expression: String?): Int {
+        require(!expression.isNullOrBlank())
+
+        val splits = expression.split(" ")
+        val numbers = splits.filterIndexed { index, _ -> index.rem(2) == 0 }.map { it.toInt() }
+        val operators = splits.filterIndexed { index, _ -> index.rem(2) == 1 }.map { Operator.of(it) }
```

Comment:
`rem` [operator 함수](https://kotlinlang.org/docs/operator-overloading.html)를 직접 호출하는 것 보다는  연산 기호로 대체해서 사용하는 것이 조금 더 가독성이 좋지 않을까요? 😃

```suggestion
        val numbers = splits.filterIndexed { index, _ -> index.rem(2) % 0 }.map { it.toInt() }
        val operators = splits.filterIndexed { index, _ -> index.rem(2) % 1 }.map { Operator.of(it) }
```

Reply:
index의 순서가 홀수, 짝수인지 기준에 따라서 `피연산자 그룹`과 `연산자` 그룹을 적절히 잘 분리해주신 것 같아요. 👍
하지만 이 코드를 처음보는 사람은 `0`과 `1`이 어떤 의미를 가지는지 바로 파악하기 힘들 것 같아요. 🤔

[1. 의미가 불분명한 매직 넘버를 상수로 선언하라.](https://javabom.tistory.com/28) 글을 참고해서, 조금 더 명확한 의미를 부여해보면 어떨까요? 🤗